airflow/lineage/__init__.py

```diff
@@ -122,12 +127,12 @@ def wrapper(self, context, *args, **kwargs):
                       for i in inlets]
             self.inlets.extend(inlets)
 
-        if len(self._inlets['datasets']) > 0:
-            self.inlets.extend(self._inlets['datasets'])
+        if self._inlets['datasets']: # pylint: disable=protected-access
+            self.inlets.extend(self._inlets['datasets']) # pylint: disable=protected-access
 
         # outlets
-        if len(self._outlets['datasets']) > 0:
-            self.outlets.extend(self._outlets['datasets'])
+        if self._outlets['datasets']: # pylint: disable=protected-access
+            self.outlets.extend(self._outlets['datasets']) # pylint: disable=protected-access
```

Comment:
There are a lot of protected accesses :/ But for now I think it is okay.

Reply:
Hi @feluelle, thanks a lot for reviewing the PR :) Yes, there are many, I opted for disabling pylint. I will fix the 1 missing space before the inline comment.